services/QuillLMS/app/controllers/feedback_history_ratings_controller.rb

```diff
@@ -0,0 +1,26 @@
+class FeedbackHistoryRatingsController < ApplicationController
+  before_action :set_feedback_history_rating, only: [:show, :edit, :update, :destroy]
+  skip_before_action :verify_authenticity_token
+
+  def create_or_update
+    rating = FeedbackHistoryRating.find_or_create_by(
```

Comment:
I think you can just do `.find_or_initialize_by`, and then you don't need to set the rating to `true` below.

Reply:
Done, good idea